docs/home/howto/CLUSTER.md

```diff
@@ -292,6 +292,35 @@ To setup the cluster do the following steps from the provisioning machine:
     epicli apply -f newcluster.yml
     ```
 
+## Note for Red Hat / CentOS Azure images
+
+Depending on your requirements you may need to create cluster based on Red Hat family (RHEL/CentOS) system. Be aware that Epiphany supports only images with RAW partitioning, and attached to standard RHEL repositories. Please read this article for more details:
+
+https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-images#rhel-7-image-types
+
+It means that actually for Azure Cloud Epiphany supports Red Hat 7.6 - 7.7 and CentOS 7.6 - 7.8 Versions (with RAW partitioning and attached to RHEL repositories).
+List of supported images will be extended in next releases.
+
+Example config for Red Hat:
+
+```
```

Comment:
Use:
'''yaml

Reply:
Also maybe make sure that its know that these changes need to be made in the infrastructure/virtual-machine documents.

Reply:
Added